packages/graphics/mesa/package.mk

```
@@ -67,8 +67,9 @@ if listcontains "${GRAPHIC_DRIVERS}" "iris"; then
   PKG_MESON_OPTS_TARGET+=" -Dintel-clc=system"
 fi
 
-if listcontains "${GRAPHIC_DRIVERS}" "(nvidia|nvidia-ng)"; then
-  PKG_DEPENDS_TARGET+=" libglvnd"
+if listcontains "${GRAPHIC_DRIVERS}" "(nvidia|nvidia-ng" ||
```

```suggestion
if listcontains "${GRAPHIC_DRIVERS}" "(nvidia|nvidia-ng)" ||
```
Close bracket missing